lib/components/atoms/Video/Video.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+/**
+ *
+ * Video
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Video.style';
+import Track from '../Track';
+import type { Props } from './types';
+
+/* eslint-disable jsx-a11y/media-has-caption */
+
+const Video = ({
+  width,
+  height,
+  autoPlay,
+  controls,
+  loop,
+  muted,
+  poster,
+  src,
+  children,
+  className,
+  controlsList,
+  defaultCaption,
+}: Props): Node => {
+  return (
+    <video
+      width={width}
+      height={height}
+      controls={controls}
+      autoPlay={autoPlay}
+      loop={loop}
+      muted={muted}
+      poster={poster}
+      className={className}
+      controlsList={controlsList}
+    >
+      <source src={src} type={`video/${src.substring(src.lastIndexOf('.') + 1)}`} />
+      {children.length > 0 ? children : <Track {...defaultCaption} />}
+    </video>
+  );
+};
+
+Video.defaultProps = {
+  width: '',
+  height: '',
+  autoPlay: false,
+  loop: false,
+  muted: false,
+  poster: '',
+  src: '',
```

Comment:
Src should be mandatory, there is no need to define it under as default

Reply:
done